src/TwigComponent/src/Twig/ComponentExtension.php

```diff
@@ -121,9 +121,9 @@ private function throwRuntimeError(string $name, \Throwable $e): void
         }
 
         if (!($e instanceof \Exception)) {
-            $e = new \Exception($e->getMessage(), $e->getCode(), $e->getPrevious());
+            $e = new \Exception($e->getMessage(), $e->getCode(), $e);
         }
```

Comment:
I believe this whole part is useless. Why converting `Throwable` to `Exception`? RuntimeError can have `Throwable` as previous parameter. Wrapping Throwables in Twig RuntimeError is correct in my opinion, but wrapping Throwables in another Exception is useless, it just adds another stacktrace and also causes the problem with lost original stack trace because of `$e->getPrevious()` error.

Reply:
I'm feeling the same, with no absolute certitude of the scenarios that probably justified this in the first place.